src/KM_Game.pas

```diff
@@ -140,9 +140,10 @@   TKMGame = class
     property IsPaused: Boolean read fIsPaused write fIsPaused;
     property MissionMode: TKMissionMode read fMissionMode write fMissionMode;
     function GetNewUID: Integer;
-    function GetDefaultMPGameSpeed: Single;
-    procedure SetDefaultMPGameSpeed(aToggle: Boolean = False);
-    procedure SetGameSpeed(aSpeed: Single; aToggle: Boolean; aToggleSpeed: Single = 1);
+    function GetNormalGameSpeed: Single;
+    function GetNormalMPGameSpeed: Single;
+    procedure SetNormalMPGameSpeed(aToggle: Boolean = False);
+    procedure SetGameSpeed(aSpeed: Single; aToggle: Boolean; aNormalSpeed: Single = 1);
```

Comment:
Ok, that's better.
1. removed
2. removed
3. there is even 1 line after refactoring
4. GetNormalGameSpeed is used in 3 places are now